07-alerting-and-notification/README.md

```diff
@@ -39,80 +43,92 @@ The following is a sample set of alerts from that list.
 
-```shell
-alertmanager:  
+```yaml
+alertmanager:
+  enabled: true
   config:
     global:
       resolve_timeout: 5m
       slack_api_url: "https://hooks.slack.com/services/TXXXX/BXXXX/<token>"
     route:
-      group_by: ['job']
-      group_wait: 30s
-      group_interval: 5m
       repeat_interval: 1h
       routes:
       - match:
-          alertname: InstanceDown
+          alertname: EmojivotoInstanceDown
         receiver: 'slack'
         continue: true
     Receivers:
-    - name: 'null'
     - name: 'slack'
       slack_configs:
-      - channel: '#promalerts'
+      - channel: '#channelName'
         send_resolved: false
         title: '[{{ .Status | toUpper }}{{ if eq .Status "firing" }}:{{ .Alerts.Firing | len }}{{ end }}] Monitoring Event Notification'
-        text: "this is a test from blueprint team from Digital Ocean"
+        text: "Emojivoto number of pods less than expected."
 ```
 
-- *slack_api_url:* When you decide to send a notification by using slack you have to create a channel that gives you a token inside of the url. We prefer here `Incoming WebHooks` for hooking. it is the easiest way to send a notification. Please visit to create a channel and get your own token from Slack. Slack messaging
-- *routes:* in this part, you will have some `receivers` and tell `routes` for the `matching receiver` here.
-- *slack_configs:* this segment includes more than 4 parts but here, `channel`, `title`, `text` are important parts for telling `Slack router` which channel and which title-text message will have. This part is totally related to how to use `notifications` in `prom-stack-values.yaml`. [Please visit the Prometheus-Notifications page for more details about this chart](https://prometheus.io/docs/alerting/latest/notification_examples/).
+Explanations for the above configuration:
 
-Now re-apply the new `prom-stack-values.yaml` using helm upgrade.Apply below upgrading command:
+- `slack_api_url` - url of the `webhook` created in step 4.
+- `routes.match.alertname` - name of the `AlertManager` alert.
+- `slack_configs` - configuration related to the channel to send the notification to, the title and the message it should include.
 
-```shell
-helm upgrade kube-prom-stack prometheus-community/kube-prometheus-stack --version 30.0.1 --namespace monitoring --create-namespace -f prom-stack-values.yaml
+Finally, upgrade the `kube-prometheus-stack`, using `Helm`:
 
-```
+  ```shell
+  HELM_CHART_VERSION="35.5.1"
+
+  helm upgrade kube-prom-stack prometheus-community/kube-prometheus-stack --version "${HELM_CHART_VERSION}" \
+    --namespace monitoring \
+    --create-namespace \
+    -f "04-setup-prometheus-stack/assets/manifests/prom-stack-values-v${HELM_CHART_VERSION}.yaml"
+  ```
 
-At this point, you should see the slack `notifications` for firing alerts.
+At this point, you should see the slack `notifications` for firing alerts when the expected number of pods for `Emojivoto` is less than 4.
```

Comment:
You are giving an example for a scenario where the emojivoto application pods count is less than 4. But, I don't see how you trigger this situation. Is there a step missing where you are downscaling the application to let's say 3 pods to simulate this scenario ?